algoliasearch/build.gradle

```diff
@@ -33,10 +33,11 @@ ext {
     - Log-in on sonatype.org, go to staging profile, select com.algolia, id = last part of URL */
 }
 
-
-nexusStaging {
-    packageGroup = PUBLISH_GROUP_ID
-    stagingProfileId = PUBLISH_STAGING_ID
+if (project.hasProperty("nexus")) {
```

Comment:
@szpak: Oh good point, moving the `nexusStaging` closure to the root `build.gradle` works and is simpler!